src/const.js

```diff
@@ -125,7 +125,7 @@ export const FLAGGED_FILE_REGEX = /thumbs\.db$|\.DS_Store$|\.orig$|\.old$|\~$/i;
 export const ALREADY_SIGNED_REGEX = /^META\-INF\/manifest\.mf/;
 
 export const FLAGGED_FILE_EXTENSION_REGEX =
-  /\.exe$|\.dll$|\.dylib$|\.so$|\.sh$|\.class$|\.swf$/i;
+  /\.exe$|\.dll$|\.dylib$|\.so$|\.sh$|\.class$|\.swf$|\.jar$/i;
```

Comment:
We should generate this regex from an array, which would be nicer for devs. Filed #1115. Obviously no need to do it in this PR.